README.md

```diff
@@ -0,0 +1,191 @@
+
+### Class fields order
+
+Fields go first. They are logically grouped in this order: constants, Activities extras, Fragments arguments, views, dependency-injected members and then plain fields. Additional subgroups can be defined.
+
+### Resources
+
+We follow the [Resources section of the Futurice's Android best practices](https://github.com/futurice/android-best-practices#resources), with the following adjustments.
+
+#### Layouts
+
+Regarding the naming, we add that list item views follow the convention of prefixing with `item`, as in `item_customer.xml`. If naming a list header (header item or header of the whole list), use the prefix `header`.
+
+Attributes with a namespace come first. Between those, first `id` (if present), then `layout_width` and then `layout_height`. Then the rest alphabetically. After this, the ones without namespace, such as `style`, alphabetically.
+
+Namespaces declarations shall be the first lines in the root element and must be ordered alphabetically. The name for `http://schemas.android.com/apk/res/android` must be `android`, `http://schemas.android.com/apk/res-auto` must be `app` and the name for `http://schemas.android.com/tools` must be `tools`.
+
+A single vertical whitespace must be left before a closing tag.
+
+The naming of the ids of the views is the following. First, if it applies, try to use Android provided ids. This applies mainly for `ListView`s, `RecyclerView`, lists items `TextView`s and so on. If it doesn't apply, the name must be the context associated, then the name and finally the view name. All snake cased, except for the view name. Example: `customer_list_description_textView`. If a layout needs identification, it's better to suffix it with just `layout`, as normally `relativeLayout` or similar won't contribute much. Also, if naming a place where a fragment will be added, name it or suffix it with `fragment_container`.
+
+#### Styles
+
+The style files should be split in different files if the size of the project is big and it deserves it. Also, use styles in views only if they are reused.
+
+The naming should be prefixed with the view name followed by a dot. Logical organization of different names (as namespaces) should be separated with dots too.
+
+#### Colors
+
+The color palette shouldn't have many elements. So it's okay for color names to be `red`, `orange` and so on (intituively inferred). If having repeated colors, use versions like `redLight`, `redLighter`, `redDark` and `redDarker`. There shouldn't be much more (it might reflect a design problem). For alpha versions of other colors, prefer the convention `red20` (being 20 the opacity in decimal). For transparent tinted versions, use something like `transparentBlackRed`.
```

Comment:
May help if here explicit tell that the alpha `20` represents a percentage, this means 20% of opacity. It is different than 20 in the scale 0...255